webapp/src/components/rhs/post_card.tsx

```diff
@@ -124,6 +124,7 @@ const PostCard = (props: Props) => {
                             timestamp: (
                                 <Timestamp
                                     value={props.post.create_at}
+                                    useRelative
```

Comment:
Linter wants this to be: 
```suggestion
                                    useRelative={true}
```

Reply:
Thanks! Done!